injected/src/features/api-manipulation.js

```diff
@@ -100,8 +100,9 @@ export default class ApiManipulation extends ContentFeature {
         if (getterValue) {
             this.wrapProperty(api, key, {
                 get: () => processAttr(getterValue, undefined),
-                enumerable: change.enumerable ?? true,
-                configurable: change.configurable ?? true,
+                // wrapProperty takes care of setting these values to the api[key] default values.
+                enumerable: change.enumerable,
+                configurable: change.configurable,
```

Comment:
this would pass undefined values, but the keys would still be there. I think you need to avoid adding the key altogether.

Reply:
Seems like an oversight on the API side tbh but thanks for the heads up :)

Reply:
Yeah maybe. I don't remember if `Object.defineProperty()` throws when it gets `enumerable: undefined`, but if it does, we should do this check inside the wrapper.